synapseclient/core/download/download_functions.py

```diff
@@ -610,6 +619,8 @@ async def download_from_url_multi_threaded(
 def download_from_url(
     url: str,
     destination: str,
+    object_id: Optional[str],
```

Comment:
Could you update this to `entity_id` and `entity_type` respectively.

Reply:
Or since `object_type` isn't always and entity, maybe `file_handle_associate_type` to match: https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/file/FileHandleAssociateType.html